export/export_handler.go

```diff
@@ -0,0 +1,91 @@
+package export
+
+import (
+	"encoding/json"
+	"fmt"
+	act "github.com/treeverse/lakefs/action"
+	"github.com/treeverse/lakefs/block"
+	"github.com/treeverse/lakefs/parade"
+)
+
+const actorName parade.ActorID = "EXPORT"
+const (
+	actionCopy   = "export-copy"
+	actionDelete = "export-delete"
+	actionNext   = "next-export"
+	actionStart  = "start-export"
+	actionDone   = "done-export"
+)
+
+type Handler struct {
+	adapter block.Adapter
+}
+
+func NewHandler(adapter block.Adapter) *Handler {
+	return &Handler{
+		adapter: adapter,
+	}
+}
+
+type TaskBody struct {
+	DestinationNamespace string
```

Comment:
We should align fields with #803, of course.  I also think we should prefer to keep some fields like in that PR.  First, because it keeps the namespace lookups more localised in code.  And secondly, it makes the message easier to read and understand -- useful when you have a bunch of broken tasks and need to understand what should have happened and didn't.
In particular, when copying I don't understand what the "destination namespace" and "destination ID" will be.  I expect the lakeFS object named `/a/b/c` to be stored in some storage namespace with a random-looking ID, but with a destination `s3://my-bucket/branch-foo-export-dir/a/b/c` -- why talk about namespaces here?